internal/utils/utils.go

```diff
@@ -393,3 +394,16 @@ func GetCollectorName(collectorType logging.LogCollectionType) string {
 	}
 	return "unknown"
 }
+
+func IsVersionAheadOrEqual(currentVersion, version string) bool {
+	if !strings.HasPrefix(currentVersion, "v") {
+		currentVersion = "v" + currentVersion
+	}
+	if version == "" {
+		return false
+	}
```

Comment:
Testing == "" after adding "v" will never pass.
```suggestion
	if version == "" {
		return false
	}
	if !strings.HasPrefix(currentVersion, "v") {
		currentVersion = "v" + currentVersion
	}
```